.github/pull_request_template.md

```diff
@@ -1,6 +1,16 @@
-## NOTICE
+## What changes were proposed in this pull request?
 
-Please create an issue in ASF JIRA before opening a pull request,
+(Please fill in changes proposed in this fix)
+
+## What is the link to the Apache JIRA
+
+(Please create an issue in ASF JIRA before opening a pull request,
 and you need to set the title of the pull request which starts with
-the corresponding JIRA issue number. (e.g. HADOOP-XXXXX. Fix a typo in YYY.)
-For more details, please see https://cwiki.apache.org/confluence/display/HADOOP/How+To+Contribute
+the corresponding JIRA issue number. (e.g. HADOOP-XXXXX. Fix a typo in YYY.
+
+Pease replace this section with the link to the original pull request)
```

Comment:
```suggestion
Please replace this section with the link to the original pull request)
```

I guess this should say "link to the JIRA issue", to match the section title.  Link to the original pull request is also useful, if moved from existing Hadoop PR.

Reply:
> Link to the original pull request is also useful, if moved from existing Hadoop PR.

Yes, Good point. Fixed to mention JIRA instead of original PR. (Original PR linking is required only during the first few days, we don't need to mention it long-term, IMHO we can just do it.)